libraries/bsg_manycore_eva.cpp

```diff
@@ -77,6 +77,32 @@
 #define DEFAULT_DRAM_BITIDX 31
 #define DEFAULT_DRAM_BITMASK (1ULL << DEFAULT_DRAM_BITIDX)
 
+static uint32_t default_vcore_max_x_coord(const hb_mc_config_t *cfg, const hb_mc_coordinate_t *tgt)
```

Comment:
I guess the min-max x-y coordinates should be within current pod, like this? https://github.com/bespoke-silicon-group/bsg_replicant/blob/main/libraries/bsg_manycore_eva.cpp#L343-L356